lib/ansible/module_utils/ec2.py

```diff
@@ -592,7 +592,7 @@ def value_is_list(my_list):
 
         # Sort list. If it's a list of dictionaries, sort by tuple of key-value
         # pairs, since Python 3 doesn't allow comparisons such as `<` between dictionaries.
-        checked_list.sort(key=lambda x: sorted(x.items()) if isinstance(x, dict) else x)
+        checked_list.sort(key=lambda x: sorted(set(x.items())) if isinstance(x, dict) else x)
```

Comment:
I tried tuple() and it doesn't work, but I'll try list. Thanks for looking at this.